From d3876d96ff2f2b00d1a6598c6ac30f329efa9d77 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Wed, 21 Nov 2018 15:27:34 +0100
Subject: [PATCH 12/12] python3: Make python version part of release number

We need to distinguish between packages targeting different python version.

Signed-off-by: Michal Hrusecky <michal.hrusecky@nic.cz>
---
 lang/python/python3-package.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lang/python/python3-package.mk b/lang/python/python3-package.mk
index 36f4ab07e..e65f9bb32 100644
--- a/lang/python/python3-package.mk
+++ b/lang/python/python3-package.mk
@@ -20,6 +20,8 @@ PYTHON3:=python$(PYTHON3_VERSION)
 
 PYTHON3PATH:=$(PYTHON3_LIB_DIR):$(STAGING_DIR)/$(PYTHON3_PKG_DIR):$(PKG_INSTALL_DIR)/$(PYTHON3_PKG_DIR)
 
+PKG_RELEASE:=$(PYTHON3_VERSION)-$(PKG_RELEASE)
+
 -include $(PYTHON3_LIB_DIR)/openwrt/Makefile-vars
 
 # These configure args are needed in detection of path to Python header files
-- 
2.27.0
